docs/explanations/status_columns.md

```diff
@@ -101,3 +101,98 @@ allow_beta: True
+  "contract": {
+    "created_at": "2000-01-02T03:04:05+06:00",
+    "id": "contract_id",
+    "name": "contract_name",
+    "products": [ "uaa-essential" ],
+    "tech_support_level": "essential"
+  },
+  "effective": null,
+  "environment_vars": [...proClientEnvironmentVariables],
+  "errors": [],
+  "execution_details": "No Ubuntu Pro operations are running",
+  "execution_status": "inactive",
+  "expires": "9999-12-31T00:00:00+00:00",
+  "features": {},
+  "machine_id": "machine_id",
+  "notices": [],
+  "result": "success",
+  "services": [
+    {
+      "available": "yes",
+      "blocked_by": [],
+      "description": "Expanded Security Maintenance for Applications",
+      "description_override": null,
+      "entitled": "yes",
+      "name": "esm-apps",
+      "status": "enabled",
+      "status_details": "Ubuntu Pro: ESM Apps is active"
+    },
+    {
+      "available": "yes",
+      "blocked_by": [],
+      "description": "Expanded Security Maintenance for Infrastructure",
+      "description_override": null,
+      "entitled": "yes",
+      "name": "esm-infra",
+      "status": "enabled",
+      "status_details": "Ubuntu Pro: ESM Infra is active"
+    },
+    {
+      "available": "yes",
+      "blocked_by": [],
+      "description": "Canonical Livepatch service",
+      "description_override": null,
+      "entitled": "yes",
+      "name": "livepatch",
+      "status": "enabled",
+      "status_details": ""
+    },
+    ...otherServiceStatusObjects
+  ],
+  "simulated": false,
+  "version": "27.13.6~18.04.1",
+  "warnings": []
+}
+```
+
+Some particularly important attributes in the output include:
+* `attached`: This boolean value indicates whether this machine is attached to an Ubuntu Pro account or not. Note that this does not imply that any particular service (e.g. esm-infra) is enabled. You must check the individual service object for that status.
+* `expires`: This is the date that the Ubuntu Pro subscription is valid until.
+* `services`: This is a list of Ubuntu Pro services. Each item has its own attributes. Widely applicable services include those with `name` equal to `esm-infra`, `esm-apps`, and `livepatch`. Some important fields in each service object are:
+  * `name`: The name of the service.
+  * `entitled`: A boolean indicating whether the attached Ubuntu Pro account is allowed to enable this service.
+  * `status`: An enum value indicating the services current status on the machine. Possible values are `enabled`, `disabled`, `n/a`, and `warning`. Any value other than `enabled` should be treated as if the service is not enabled and not working properly on the machine.
+
+For example, if you want to programatically determine if esm-infra is enabled and working on a particular machine, you can use the following command:
+```shell
+sudo pro status --format=json | jq '.services[] | select(.name == "esm-infra").status == "enabled"'
+```
+That command will print `true` if esm-infra is enabled, or `false` if it is not.
+
```

Comment:
Side question - what should they do if it's disabled and they expected it to be enabled and working?

Reply:
It depends on what the value is.
If it's "disabled", then run `pro enable`.
If it's `n/a` then you don't have the option to enable it.
If it's `warning` then something specific is wrong and you need to go read the warning detailed message to figure out what to do.
I'm not sure if all of that belongs here or not - I could be convinced either way - what do you think?

Reply:
I might have misunderstood, but the command reads to me like you're only checking if it's enabled (or not) - does the output actually tell you more than that whether it's available-but-disabled, n/a, or broken? So my initial question was more along the lines of "what would they need to follow up with if they get "false" as the result?"

Reply:
I think I see your concern now. I've modified the example and the preceding text a bit. LMK what you think

Reply:
Makes much more sense! Thanks for changing it :)